src/lib/templateengine.js

```diff
@@ -687,7 +703,7 @@ function TemplateEngine( undefined ) {
       if (!ret && m['defaultValue']) {
         ret = mapValue(m['defaultValue']);
       }
-      if (ret) {
+      if( ret !== undefined ) {
```

Comment:
As this is included here and not on the original version it seems that this patch is against a old version. (GitHub tells me that it is CometVisu/CometVisu develop but that's not correct when you refer to https://github.com/CometVisu/CometVisu/blame/develop/src/lib/templateengine.js )
=> Can you rebase that request on the latest CometVisu/CometVisu develop?